pennylane_lightning/core/src/observables/tests/Test_Observables.cpp

```diff
@@ -207,6 +207,29 @@ template <typename TypeList> void testHermitianObsBase() {
             REQUIRE(ob2 != ob3);
         }
 
+#ifdef PL_USE_LAPACK
+        DYNAMIC_SECTION("Failed to create a HermitianObs- "
+                        << StateVectorToName<StateVectorT>::name) {
+            std::mt19937_64 re{1337};
+            const size_t num_qubits = 3;
```

Comment:
also in the other test units:
```suggestion
            constexpr size_t num_qubits = 3;
```